testsuite/integration/clustering/pom.xml

```diff
@@ -214,6 +214,29 @@
                     </plugin>
                 </plugins>
             </build>
+            <dependencies>
+                <dependency>
+                    <groupId>org.jboss.byteman</groupId>
```

Comment:
I don't know if we want to include byteman just to get extra debugging info on failure. Ideally all this stuff would be logged at debug or trace anyway, so you could get more info by changing the log level.

Reply:
byteman has been removed